src/lib/web.js

```diff
@@ -531,9 +531,9 @@
 		}
 	});
 
-	function putInto(context, prop, valueToPut) {
+	function putInto(runtime, context, prop, valueToPut) {
```

Comment:
Comment that's outside the scope (ha) of this PR: Does the runtime have any state? If not, why not have top-level functions instead of passing the runtime around?